docs/docs/glamor.md

```diff
@@ -33,7 +33,19 @@ module.exports = {
 
 Then in your terminal run `gatsby develop` to start the Gatsby development server.
 
-Now let's create a sample Glamor page at `src/pages/index.js`
+Now let's create a new `Container` component.
+Create a `components` directory at `src/components` and then, in this directory,
+create a file named `container.js` and paste the following:
+
+```javascript
+import React from "react";
+
+export default ({ children }) => (
+  <div style={{ margin: "3rem auto", maxWidth: 600 }}>{children}</div>
```

Comment:
Shouldn't we be using Glamor here?

Reply:
You are right. I just copy-pasted the component from the part 2 tutorial.
It make sense to use Glamor for glamor tutorial, and Styled Components for the styled components one.
I'll work on it!

Sorry for this :sweat_smile:

Reply:
Ah, shit. Sorry for the premature merge. 🤦‍♂️